superset/assets/src/dashboard/components/SaveModal.jsx

```diff
@@ -16,6 +15,7 @@ const propTypes = {
   dashboardTitle: PropTypes.string.isRequired,
   expandedSlices: PropTypes.object.isRequired,
   layout: PropTypes.object.isRequired,
+  saveType: PropTypes.string,
```

Comment:
if you add a string prop you should _always_ enumerate all valid options.

Reply:
add enumeration.